sdk/exportentities/workflow_test.go

```diff
@@ -218,13 +221,36 @@ func TestWorkflow_GetWorkflow(t *testing.T) {
 		PipelineHooks          []exportentities.HookEntry
 		Permissions            map[string]int
 		HistoryLength          int64
+		OneAtATime             *bool
 	}
 	tsts := []struct {
 		name    string
 		fields  fields
 		want    sdk.Workflow
 		wantErr bool
 	}{
+		// pipeline
+		{
+			name: "Simple workflow with mutex should not raise an error",
+			fields: fields{
+				PipelineName: "pipeline",
+				OneAtATime:   &strue,
+			},
+			wantErr: false,
+			want: sdk.Workflow{
+				HistoryLength: sdk.DefaultHistoryLength,
+				WorkflowData: &sdk.WorkflowData{
+					Node: sdk.Node{
+						Name: "pipeline",
+						Type: "pipeline",
+						Context: &sdk.NodeContext{
+							PipelineName: "pipeline",
+							Mutex:        true,
+						},
+					},
+				},
+			},
+		},
```

Comment:
i miss a test with several pipelines

Reply:
already done on `Complexe workflow without joins and mutex should not raise an error`